AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/UI/ALMAXSwiftUITemplateNativeAdView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  ALMAXSwiftUITemplateNativeAdView.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Matthew Nguyen on 8/1/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import SwiftUI
+import Adjust
+import AppLovinSDK
+
+@available(iOS 13.0, *)
+struct ALMAXSwiftUITemplateNativeAdView: View
+{
+    @ObservedObject private var viewModel = ALMAXSwiftUITemplateNativeAdViewModel(adUnitIdentifier: "YOUR_AD_UNIT_ID")
+    
+    var body: some View {
+        VStack {
+            ALMAXNativeTemplateAdViewSwiftUIWrapper(shouldShowAd: $viewModel.triggerShowAd,
+                                                 nativeAdLoader: viewModel.nativeAdLoader,
+                                                 containerView: viewModel.containerView,
+                                                 didLoadNativeAd: viewModel.didLoadNativeAd(_:for:),
+                                                 didFailToLoadNativeAd: viewModel.didFailToLoadNativeAd(forAdUnitIdentifier:withError:),
+                                                 didClickNativeAd: viewModel.didClickNativeAd(_:),
+                                                 didExpireNativeAd: viewModel.didExpireNativeAd(_:),
+                                                 didPayRevenue: viewModel.didPayRevenue(for:))
```

Comment:
Indentation nit, arguments should be at the same indentation over multiple lines. If you highlight and do Ctrl+I, Xcode should fix it.